nexus/types/src/inventory/display.rs

```diff
@@ -895,6 +896,32 @@ fn display_sleds(
             }
         }
 
+        if !health_monitor.is_empty() {
```

Comment:
I think it'd be worth reporting explicitly when no services were found in maintenance.

Reply:
Yeah, that makes sense. Do you mind if I make this visible with the follow up PR that adds the types to the DB? Otherwise, this will always show up as if everything was healthy and it could be misleading. I'll add a comment explaining for now.